app/src/main/res/values/strings.xml

```diff
@@ -133,6 +131,8 @@
     <string name="preferences_language">Language</string>
     <!-- Preference for data choices -->
     <string name="preferences_data_choices">Data choices</string>
+    <!-- Preference linking to the privacy notice -->
+    <string name="preferences_privacy_notice">Privacy notice</string>
```

Comment:
Sorry, one more nit here: we will actually need to change the string key `preferences_privacy_notice` to something else so our translators see that a change was made. Perhaps something like `preferences_privacy_link`

Reply:
Should I also change the label to "Privacy link"?

Reply:
The string should stay the same, just need the "name" portion to change :)

Reply:
Done.